scripts/create_sample_worksheet.py

```diff
@@ -370,6 +558,11 @@ def main():
         action='store_true',
         help='Whether to clean up bundles and worksheets created from previous sample worksheets (defaults to false)',
     )
+    parser.add_argument(
+        '--test',
```

Comment:
Instead of calling this `test`, maybe should be more specific - `test-print` or something.